DOC-852: Credentials overhaul A-D (#2086)
n8n-io/n8n-docs: docs/integrations/builtin/credentials/cal.md

Create a [Cal](https://www.cal.com/) pro account.

## Supported authentication methods

- API key

## Related resources

Refer to [Cal's API documentation](https://cal.com/docs/enterprise-features/api#api-server-specifications){:target=_blank .external-link} for more information about the service.

## Using API key

To configure this credential, you'll need:

- An **API Key**: Refer to the [Cal API Quick Start documentation](https://cal.com/docs/enterprise-features/api/quick-start){:target=_blank .external-link} for information on how to generate a new API key.

/// note | Hosted API

> Cal previously supported [a hosted API](https://cal.com/docs/enterprise-features/api/quick-start/hosted-api-through-cal-com){:target=_blank .external-link}, but they have since discontinued that service. You do not need to make any edits to the **Host** field within the credential.

Reviewer: ```suggestion
Cal previously supported [a hosted API](https://cal.com/docs/enterprise-features/api/quick-start/hosted-api-through-cal-com){:target=_blank .external-link}, but they have since discontinued that service. You don't need to make any edits to the **Host** field within the credential.
```
